index.md

```diff
@@ -140,18 +140,17 @@ Chain investigations in 9/10 player games are almost always a bad idea. They lim
 
 ### Executions
 #### As a Liberal
-If you are not, then you need to be careful not to give the Fascists an even player count to the Liberals so that they can’t mass downvote good governments. This is particularly important in games with an odd number of players, where killing a liberal by accident will make it hard for the liberals to win the game. It can often be safer to kill someone you *know* is Fascist rather than fishing for Hitler and killing a liberal by accident; particularly because Hitler will often be playing as a Liberal.
+You must be careful when you have gained the power of Assassination. The Liberals only have a narrow majority, as such you must make sure you do not shoot one of your fellow Liberals. There are two real choices to make as a Liberal with the Assassination power: shoot a known Fascist or try to shoot Hitler.
 
-It is difficult to identify Hitler over the course of the game. If you have suspicion of someone for being Hitler you should be sure before assassinating as Hitler could be disguised as the most Liberal player instead.
+There are only two ways to know for sure if someone is fascist:
+* Investigation
+* Be in direct dispute over a policy
 
-For identifying Hitler for assassination, semi-valid reasons for assassinations can include but are not limited to the following:
+It is often safest to shoot whoever you are in a dispute with as a result of investigations of policy disputes. They may be less likely to be Hitler and this may not grant you much trust with others but the Liberals majority will increase. The alternative is to aim for someone else with the aim of shooting Hitler. The hard part is identifying them. The following reasons are possible reasons you may try to shoot Hitler:
```

Comment:
True, in a bunch of situations where I have been in that position I became untrusted.

If you have gotten into a conflict it is *likely* not Hitler. So based on that

| Players | Other Fascist + Hitler | Chance of killing Hitler | Chance of killing another Fascist (who is not Hitler) |
| --- | --- | --- | --- |
| 5 | 1 | 33.33% | 33.33% |
| 6 | 1 | 25 % | 0-25% |
| 7 | 2 | 20% | 20-40% | 
| 8 | 2 | 16% | 16.67-33.33% |
| 9 | 3 | 14% | 28.57-42.86% |
| 10 | 3 | 11 % | 25-37.5% |

For the 5 player game, Hitler might be more outlandish